src/dra/75F54.c

```diff
@@ -174,7 +177,9 @@ void func_80116408(void) {
     case 4:
         if (PLAYER.animFrameIdx == 10 && PLAYER.animFrameDuration == 1) {
             g_Player.D_80072F18 = 4;
-            CreateEntFactoryFromEntity(g_CurrentEntity, 0x25U, 0);
+            // Make factory with blueprint 37. This creates entity with ID 28,
+            // which is EntityExpandingCircle.
+            CreateEntFactoryFromEntity(g_CurrentEntity, 37, 0);
```

Comment:
The issue is that the blueprint number doesn't match the Entity ID. Blueprint 37 creates entity 28, so even if we populate 28 being EntityExpandingCircle, that wouldn't connect to the factory blueprint. We could consider making a new enum for the blueprint numbers and call it BlueprintIDs, but it would be distinct from EntityIDs.